### ISSUE TITLE
Add Estimated Credits when using Quick Visualization

### ISSUE THREAD

**Issue by vatz88** [vatz88/FFCSonTheGo#235]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
http://disq.us/p/2wcary2

**Comment by therealsujitk** [comment]
This isn't always true, not sure if it's even do-able. For my batch atleast, we've never had decimal credits, it's something new they've introduced for juniors, plus if I remember correctly all of my courses were either 1 credit, 3 credits or 4 credits. As far as i've seen there's no proper relation between slots and credits (number of slots, maybe, but not the slots itself), but maybe they're trying to do something for new batches.

**Comment by vatz88** [comment]
What I wrote was based on the disqus thread linked.

The person is probably referring to this. When you have `A1` slot selected, you know it's 2 credits. `A1+TA1` should be 3 credits.
<img width="635" alt="image" src="https://github.com/vatz88/FFCSonTheGo/assets/16202323/b591cb82-de97-4eeb-b089-0232f61b891e">

**Comment by AnantGupta001** [comment]
yeah it's true. Number of slots is directly proportional to the number of credits.
